vmdb/spec/controllers/ops_settings_spec.rb

```diff
@@ -88,5 +88,50 @@
         @sch.should_not be_enabled
       end
     end
+
+    context "schedule additon" do
+      before(:each) do
+        EvmSpecHelper.create_guid_miq_server_zone
+        session[:userid] = User.current_user.userid
+        controller.should_receive(:render)
+        @schedule = FactoryGirl.create(:miq_schedule, :userid => "test", :towhat => "Vm")
+        @params = {
+          :button             => "add",
+          :description        => "description01",
+          :enabled            => "on",
+          :filter_typ         => "all",
+          :action_typ         => "vm",
+          :timer_typ          => "Once",
+          :time_zone          => "UTC",
+          :start_hour         => "0",
+          :start_min          => "0",
+          :miq_angular_date_1 => (Time.now + 60 * 60 * 24).strftime("%m/%d/%Y")
+        }
+        controller.stub(:assert_privileges)
+      end
+
+      after(:each) do
+        expect(response.status).to eq(200)
+      end
+
+      it "#dont_allow_duplicate_names_when_adding" do
```

Comment:
This syntax for the it statement is for when you have a method named `dont_allow_duplicate_names_when_adding`.  I don't see a method like that, so I think this should just be a normal English string.